spec/support/schemas/stores/index.json

```diff
@@ -0,0 +1,94 @@
+{
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "$ref": "#/definitions/Root",
+  "definitions": {
+    "Root": {
+      "type": "object",
+      "additionalProperties": false,
+      "properties": {
+        "own_stores": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Store"
+          }
+        },
+        "spouse_stores": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Store"
+          }
+        }
+      },
+      "required": ["own_stores", "spouse_stores"],
+      "title": "Root"
+    },
+    "Store": {
```

Comment:
It's really cool and, I think, nice that https://jsonformatter.org/json-to-jsonschema is able to extract named type definitions (and, in turn, this type references another extracted `Item` type), rather than just building a single, big, deeply nested blob.